umap/utils.py

```diff
@@ -109,3 +109,33 @@ def gzip_file(from_path, to_path):
     with open(from_path, 'rb') as f_in:
         with gzip.open(to_path, 'wb') as f_out:
             f_out.writelines(f_in)
+
+def user_details(strategy, details, user=None, *args, **kwargs):
+    """Update user details using data from provider."""
+    if not user:
+        return
+
+    changed = False  # flag to track changes
+    # uMap fix: remove 'username' from protected fields
+    protected = ('id', 'pk', 'email') + \
```

Comment:
Maybe those protected fields could be another setting upstream ?

Reply:
Yes, I want to make a PR to social_auth that way :+1: